efinpro/Controllers/HomeController.cs

```diff
@@ -35,14 +38,33 @@ public ActionResult Index(Users _userdetails)
             {
                 UserMailID = _userdetails.MailID;
                 Password = _userdetails.Password;
```

Comment:
No additional variable when there is no change in the object data

Reply:
Removed additional variables.